zktop.py

```diff
@@ -93,14 +95,14 @@ def __init__(self, session, server_id):
 class ZKServer(object):
     def __init__(self, server, server_id):
         self.server_id = server_id
-        self.host, self.port = server.split(':')
+        self.host, self.port = server.split(':') if ':' in server else (server, ZK_DEFAULT_PORT) # fallback to default if user doesn't specify port number
         try:
-            stat = send_cmd(self.host, self.port, b'stat\n')
+            stat = send_cmd(self.host, int(self.port), b'stat\n')
```

Comment:
make port an integer as part of parsing, rather than having to do this multiple times in the code.